Gemfile.lock

```diff
@@ -16,8 +16,8 @@ GEM
       ast (~> 2.4.1)
     rainbow (3.0.0)
     rake (13.0.6)
-    redis (4.4.0)
-    redlock (1.2.1)
+    redis (4.5.1)
+    redlock (1.2.2)
```

Comment:
Should this gems version update happen in an other PR?

Reply:
Yeah could be too, that was automatic but yes, I'll update them separately.